contracts/PersonalInvite.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity 0.8.17;
 
 import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/math/Math.sol";
```

Comment:
warum brauchen wir das wenn es vorher nicht drin war?

Reply:
It has to be caused by the switch from openzeppelin contracts to openzeppelin contracts upgradeable in Token.sol. I looked for the specific inheritance but can't find it. It is required or the project won't compile. 
Is this answer good enough? Otherwise I will investigate deeper, but I already spent 30 minutes on this.